docs/configuring_tor.md

```diff
@@ -133,6 +136,24 @@ specification of an additional argument:
 ⛰  ./lnd --tor.active --tor.streamisolation
 ```
 
+## Authentication
+
+In order for `lnd` to communicate with the Tor daemon securely, it must first
+establish an authenticated connection. `lnd` supports the following Tor control
+authentication methods (arguably, from most to least secure):
+
+* `SAFECOOKIE`: This authentication method relies on a cookie created and
+  stored by the Tor daemon and is the default assuming the Tor daemon supports
+  it by specifying `CookieAuthentication 1` in its configuration file.
+* `HASHEDPASSWORD`: This authentication method is stateless as it relies on a
```

Comment:
Maybe make a note that the password should be the output of `tor --hash-password "password here"`